src/Integration/Resources/Strings.Designer.cs

```diff
@@ -701,24 +701,6 @@ public static string QualityProfileDownloadFailedMessageFormat {
             }
         }
         
-        /// <summary>
-        ///   Looks up a localized string similar to Refresh.
-        /// </summary>
-        public static string RefreshCommandDisplayText {
```

Comment:
quite a few more of these seem to no longer be used. could you clean them up too?